docs: Add npm strictSSL field (#2882)
saucelabs/sauce-docs: docs/web-apps/automated-testing/cucumberjs-playwright/yaml.md

---

### `packages`

<p><small>| OPTIONAL | OBJECT |</small></p>

Specifies any npm packages that are required to run tests and should, therefore, be installed on the Sauce Labs VM. See [Including Node Dependencies](#including-node-dependencies).

```yaml
npm:
  packages:
    lodash: "4.17.20"
    "@babel/preset-typescript": "7.12"
```

:::caution
Do not use `dependencies` and `packages` at the same time.
:::

---

### `strictSSL`

<p><small>| OPTIONAL | BOOLEAN |</small></p>

> Instructs npm to perform SSL key validation when making requests to the registry via HTTPS (`true`) or not (`false`). Defaults to `true` if not set.

Reviewer: I think it normally defaults to `false`. It's only in the cli driven case it defaults to `true.

Author: Nice catch.

:::note

> When running tests and installing packages via a Sauce Connect tunnel, it is required to set `strictSSL` to `false`.

Reviewer 2: ```suggestion
If you're using a Sauce Connect tunnel, you must set `strictSSL` to `false`.
```
